### ISSUE TITLE
Missing parentheses in LaTeX / pprint output

### ISSUE THREAD

**Issue by iamthad** [sympy/sympy#13651]
There appears to be a bug printing LaTeX representations of sums containing sums multiplied by -1:
```pycon
>>> import sympy
>>> sympy.__version__
'1.1.1'
>>> from sympy.abc import a, b, c
>>> buggy = c + sympy.collect_const(-a - b)
>>> buggy
c - (a + b)
>>> sympy.srepr(buggy)
"Add(Symbol('c'), Mul(Integer(-1), Add(Symbol('a'), Symbol('b'))))"
>>> sympy.latex(buggy)
'c - a + b'
```

EDIT:
This also occurs with `pprint`:
```pycon
>>> sympy.pprint(buggy)
c - a + b
```

**Comment by cbm755** [comment]
I feel like this might be dupe but I cannot find a reference...  Confirmed on master branch.

**Comment by amitmanchanda1995** [comment]
@cbm755 I interested in solving this bug. Could you please help me on how to start?
Thanks

**Comment by Subhash-Saurabh** [comment]
Hey @amitmanchanda1995  I have already started working on this problem.

**Comment by cbm755** [comment]
Anyone can work on an issue.  Send PRs if you make progress.

I can't really help except to say read the latex printer code and play with it.

**Comment by asmeurer** [comment]
A more direct way to create the expression is `c + Mul(-1, a + b, evaluate=False)`.

**Comment by gxyd** [comment]
Closing. Fixed by #13673 .
